### PR TITLE
Update filestructure

### PR DESCRIPTION
**Pull request by c0derMo**
Updates the directory-structure to group individual commands, tasks, listeners, etc. by their function instead of type.
Also removes a couple files, merges some files and adds CODEOWNERS, to use in future pullrequests.

### CODE REVIEW

File: modules/attendancetracker/attendanceTrackerButton.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Interaction } from "discord.js";
```
Comment (Kr0nox) [inline modules/attendancetracker/attendanceTrackerButton.ts:1]: Rename to ...ButtonListener or ...ButtonEventHandler

File: modules/mensa/mensaCommand.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import {ISlashCommand} from "../types";
```
Comment (Kr0nox) [inline modules/mensa/mensaCommand.ts:1]: Rename to ...MenuListener or ...MenuEventHandler

**Review by Kr0nox — CHANGES_REQUESTED** [review]
I would suggest renaming the given files and additionally rename truthtable.ts to truthtableParser.ts, to clearly show what they are doing

File: modules/mensa/mensaCommand.ts (hunk 2 above)
Comment (c0derMo) [reply modules/mensa/mensaCommand.ts:1]: Assuming you meant the koeriSelectionMenu, closing this

**Review by Kr0nox — APPROVED** [review]
lgtm

**Pull request merged**